### PR TITLE
Teachers/learning

### PR DESCRIPTION
**Pull request by HectorOF**
translated folder techers/learning to sapanish

### CODE REVIEW

**Comment by bryanmg** [comment]
For the next time, please use english for description of your commits 👍

File: app/views/teachers/learning_resources/_learning_resource.html.erb [hunk 1]
```diff
@@ -8,24 +8,24 @@
         </strong>
       </p>
       <p>
-        <strong>Description:</strong>
+        <strong>Descripción:</strong>
```
Comment (bryanmg) [inline app/views/teachers/learning_resources/_learning_resource.html.erb:11]: You forgot the word 'name' in the line 7 of this file

Comment (cesargomez89) [reply app/views/teachers/learning_resources/_learning_resource.html.erb:11]: No sean cabrones use I18n
```erb
<strong><%= t('.description')%</strong>
```
https://guides.rubyonrails.org/i18n.html

Y ya saquenme de aqui @bryanmg, diario me llegan un chingo de notificaciones. les desactive los github actions btw jjjjjj.
Saludos :heart:

**Review by bryanmg — APPROVED** [review]
LGTM!

**Pull request merged**